tools/shoestring/tests/commands/test_reset_data.py

```diff
@@ -21,11 +21,11 @@ def _create_directories_with_placeholders(directory, subdirectory_names):
 
 # region basic
 
-async def _assert_reset_data(node_features, expected_recreated_subdirectories):
+async def _assert_reset_data(node_features, expected_recreated_subdirectories, light_api=False):
```

Comment:
I would probably use ``full_api`` instead of ``light_api`` to match the dev code.
Not sure if this is the value read from the config.

Reply:
> Not sure if this is the value read from the config.

yes, `prepare_shoestring_configuration` method setup config is reading from `sai.ini`.